eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -129,6 +129,7 @@ jobs:
         Windows_NetCoreApp:
           OSVmImage: "windows-2019"
           TestTargetFramework: netcoreapp2.1
+          CodeCoverageArgs: /p:CollectCoverage=true /p:CoverletOutputFormat=cobertura
```

Comment:
Should we default CoverletOutputFormat=cobertura in the targets/props file instead of needing to explicitly pass it? Ideally we would only need to set the one property to enable coverage.